examples/example_glfw_wgpu/README.md

```diff
@@ -1,4 +1,38 @@
-## How to Build
+## How to Build     
+
+### Using CMake
+#### Building for desktop (WebGPU-native) with Google Dawn:
+ 1. `git clone https://github.com/google/dawn dawn`
+ 2. `cmake -B build -DIMGUI_DAWN_DIR=dawn`
+ 3. `cmake --build build`
+The resulting binary will be found at one of the following locations:
+  * build/Debug/example_sdl2_wgpu[.exe]
+  * build/example_sdl2_wgpu[.exe]
+
+#### Building for desktop (WebGPU-Native) with WGPU:
+ 1. download WGPU-Native autogenerated binary modules for your platform/compiler from: https://github.com/gfx-rs/wgpu-native/releases
+ 2. unzip the downloaded file in `your_preferred_folder`
+ 3. move into `your_preferred_folder` (e.g. typing: `cd your_preferred_folder`)
+ 4. `cmake -B build -DIMGUI_WGPU_DIR=your_preferred_folder`  ("full path" or "relative" starting from current directory)
+ 5. `cmake --build build`
+The resulting binary will be found at one of the following locations:
```

Comment:
These instructions are misleading. Step 3 should be: move to `examples/example_glfw_wgpu`. `your_preferred_folder` is the folder in which you unzipped WGPU-Native